tests-utils/src/main/scala/tech/beshu/ror/utils/containers/images/ReadonlyRestPlugin.scala

```diff
@@ -66,16 +66,18 @@ class ReadonlyRestPlugin(esVersion: String,
                          performPatching: Boolean)
   extends Elasticsearch.Plugin {
 
-  override def updateEsImage(image: DockerImageDescription): DockerImageDescription = {
+  override def updateEsImage(image: DockerImageDescription, esConfig: Elasticsearch.Config): DockerImageDescription = {
     image
       .copyFile(os.root / "tmp" / config.rorPlugin.name, config.rorPlugin)
-      .copyFile(configDir / "ror-keystore.jks", fromResourceBy(name = "ror-keystore.jks"))
-      .copyFile(configDir / "ror-truststore.jks", fromResourceBy(name = "ror-truststore.jks"))
-      .copyFile(configDir / "elastic-certificates.p12", fromResourceBy(name = "elastic-certificates.p12"))
-      .copyFile(configDir / "elastic-certificates-cert.pem", fromResourceBy(name = "elastic-certificates-cert.pem"))
-      .copyFile(configDir / "elastic-certificates-pkey.pem", fromResourceBy(name = "elastic-certificates-pkey.pem"))
-      .updateFipsDependencies()
-      .copyFile(configDir / "readonlyrest.yml", config.rorConfig)
+      .copyFile(esConfig.esConfigDir / "ror-keystore.jks", fromResourceBy(name = "ror-keystore.jks"))
+      .copyFile(esConfig.esConfigDir / "ror-truststore.jks", fromResourceBy(name = "ror-truststore.jks"))
+      .copyFile(esConfig.esConfigDir / "elastic-certificates.p12", fromResourceBy(name = "elastic-certificates.p12"))
+      .copyFile(esConfig.esConfigDir / "elastic-certificates-cert.pem", fromResourceBy(name = "elastic-certificates-cert.pem"))
+      .copyFile(esConfig.esConfigDir / "elastic-certificates-pkey.pem", fromResourceBy(name = "elastic-certificates-pkey.pem"))
+      .updateFipsDependencies(esConfig)
+      .user("elasticsearch")
+      .copyFile(esConfig.esConfigDir / "readonlyrest.yml", config.rorConfig)
+      .user("root")
```

Comment:
We don't have to install the plugin as the root user. But we have to patch it as root.